parent/pom.xml

```diff
@@ -2883,11 +2883,7 @@
             <groupId>org.apache.maven.plugins</groupId>
             <artifactId>maven-surefire-plugin</artifactId>
             <configuration>
-              <excludes combine.children="append">
-                <exclude>**/SnapshotChecksumTest.java</exclude>
-                <!-- Re-add the default exclude for inner classes, see https://github.com/camunda/camunda/issues/8637 -->
-                <exclude>**/*$*.java</exclude>
-              </excludes>
+              <groups>!strace</groups>
```

Comment:
:x: You can exclude them by default in the main configuration. See where we set `!performance` on line 2346 of the parent/pom. Then we don't need the skip-strace-tests profile at all.

Reply:
Indeed, and I would have been blocked to concat/merge `<groups>!performance</groups>` and `<groups>!strace</groups>` anyway.

Reply:
Thx for pairing on this 👍